packages/bpk-component-calendar/src/Week.js

```diff
@@ -205,39 +205,41 @@ class Week extends Component {
 
     return (
       <tr className={getClassName('bpk-calendar-grid__week')}>
-        {this.props.dates.map(date => (
-          <DateContainer
-            className={this.props.cellClassName}
-            isEmptyCell={!isSameMonth(date, month) && ignoreOutsideDate}
-            key={date.getDate()}
-            weekendStart={
-              showWeekendSeparator && firstDayOfWeekendIndex === getDay(date)
-            }
-            weekendEnd={
-              showWeekendSeparator && lastDayOfWeekendIndex === getDay(date)
-            }
-          >
-            <DateComponent
-              date={date}
-              modifiers={dateModifiers}
-              aria-label={formatDateFull(date)}
-              onClick={onDateClick}
-              onDateKeyDown={onDateKeyDown}
-              preventKeyboardFocus={preventKeyboardFocus}
-              isKeyboardFocusable={isKeyboardFocusable}
-              isFocused={isSameDay(date, focusedDate)}
-              isSelected={isSameDay(date, selectedDate)}
-              isBlocked={
-                minDate && maxDate
-                  ? !isWithinRange(date, minDate, maxDate)
-                  : false
+        {this.props.dates.map(date => {
+          const isBlocked =
+            minDate && maxDate ? !isWithinRange(date, minDate, maxDate) : false;
+
+          return (
+            <DateContainer
+              className={this.props.cellClassName}
+              isEmptyCell={!isSameMonth(date, month) && ignoreOutsideDate}
+              key={date.getDate()}
+              weekendStart={
+                showWeekendSeparator && firstDayOfWeekendIndex === getDay(date)
+              }
+              weekendEnd={
+                showWeekendSeparator && lastDayOfWeekendIndex === getDay(date)
               }
-              isOutside={markOutsideDays && !isSameMonth(date, month)}
-              isToday={markToday && isToday(date)}
-              {...dateProps}
-            />
-          </DateContainer>
-        ))}
+            >
+              <DateComponent
+                date={date}
+                modifiers={dateModifiers}
+                aria-hidden={isBlocked}
+                aria-label={isBlocked ? '' : formatDateFull(date)}
```

Comment:
Date change 2 - apply here as the Week can have any custom date component passed in.

Make `isBlocked` a const and use in `aria-hidden` and `aria-label`.